admin/howto/create-billing-account.md

```diff
@@ -0,0 +1,73 @@
+# Provision a paid cloud provider account for 2i2c
+
+For 2i2c to build & manage your JupyterHub, it needs *full* access to a
+cloud provider account on [Google Cloud Platform](https://cloudn.google.com),
+[Amazon Web Services][https://aws.amazon.com], or [Azure](https://azure.microsoft.com).
+This page will provide guidance on how to do that.
+
+## Google Cloud Platform
+
+Currently, Google Cloud Platform (GCP) is 2i2c's preferred cloud provider. 2i2c
+access can be set up in a couple different ways.
+
+### Full billing account access
+
+A GCP [billing account](https://cloud.google.com/billing/docs/how-to/manage-billing-account)
+is attached to a source of funds (Credit Card, Institutional account with invoicing,
+credits provided by Google, etc). Giving 2i2c full access to a billing account
+lets us manage everything except the funding mechanism. You can track costs,
+get invoices, and grant overall access by controlling the billing account.
+
+1. [Create a billing account](https://cloud.google.com/billing/docs/how-to/manage-billing-account#create_a_new_billing_account)
+   This should be fairly straightforward with a credit card.
+   
+2. Give 2i2c [billing administrator](https://cloud.google.com/billing/docs/how-to/billing-access)
+   permissions. You can do this by:
+   
+   1. Going to 'Account Management' on the left menu inside your billing account
```

Comment:
@yuvipanda, do you think adding screenshots will make sense here? I highly appreciate instructions that provide screenshots because I can double check that I'm the right place.

If the menu is super obvious, please ignore the noise :D

Reply:
agree that screenshots would help - though probably don't need to be a blocker for the PR if Yuvi would just like to get something in

Reply:
Agree re: screenshots. I don't want to block this PR for screenshots tho